packages/new_stats/src/components/stats_table.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <!--loading-->
+    <v-sheet color="transparent" height="82vh" v-if="loading" class="d-flex align-center justify-center">
+      <v-container class="text-center d-block">
+        <v-progress-circular size="40" indeterminate />
+        <p class="pt-4 font-weight-bold">Loading Statistics</p>
+      </v-container>
+    </v-sheet>
+
+    <!--error-->
+    <v-sheet color="transparent" height="82vh" v-else-if="failed" class="d-flex align-center w-100 justify-center">
+      <v-container class="text-center">
+        <v-icon color="error" size="x-large">mdi-close-circle-outline</v-icon>
+        <v-container class="text-error">
+          Failed to get stats data, Please check you internet connection or try again later</v-container
+        >
+        <v-btn class="text-capitalize" @click="getStatsData(true)" color="secondary">Try again </v-btn>
```

Comment:
Add disabled attribute to the button if props.networks haven't been passed yet.

Reply:
there is no way for this to happen, the initial stat of networks [dev,test,main] and the user can't switch off more than two toggles